moz/main/views.py

```diff
@@ -1,19 +1,26 @@
-from flask import render_template, Blueprint, current_app
+import os
+
+from flask import render_template, Blueprint, current_app, send_from_directory
 from flask_login import login_required
 
+from config import MEDIA_URL, MEDIA_ROOT
```

Comment:
Why not to use current_app.config["KEY"] to reference config file only once when importing config in __init__?

Reply:
cant use it inside @main.route as its initilized before current_app context exist